installer/lib/phx_new/app.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Mix.Tasks.Phx.New.App do
+  use Mix.Task
+  use Mix.Tasks.Phx.New.{Project, Generator}
+
+  @prefix "phx_umbrella/apps/app_name"
+
+  template :new, [
+    {:eex, "#{@pre}/config/config.exs",      "apps/app_name/config/config.exs"},
```

Comment:
@OvermindDL1 this was a 2am checkpoint commit last night. Untested and not currently building. Thanks for taking a look! :)